docs/INSTALL.md

```diff
@@ -79,71 +118,58 @@ export PATH="$PATH:/home/plume/.cargo/bin:/home/plume/.local/bin:/usr/local/sbin
 
 On **Windows**, you'll need, if you don't already have them, to download and install the [Visual C++ 2015 Build Tools](https://www.microsoft.com/en-us/download/details.aspx?id=48159). Then, download the [rustup installer](https://www.rust-lang.org/en-US/install.html) and run it.
 
-## Getting and compiling the Plume source code
+## Getting Plume's source code
 
-Plume needs to be compiled from source.
+Plume needs to be compiled from source. To download the code, run:
 
 ```bash
 git clone https://github.com/Plume-org/Plume.git
 cd Plume
 
-# This may take some time as RustUp will download all
-# the required Rust components, and Cargo will download
-# and compile all dependencies.
-cargo build
-```
-
-We may provide precompiled packages and Docker images in the future; if you have experience in these fields and want to help, feel free to discuss this in issues and to propose pull-requests!
-
-## Configuring PostgreSQL
-
-You can either run PostgreSQL from the machine that runs Plume, or from another server. We recommend you to use the first setup for development environments, or in production for small instances.
-
-In the first case, just run this command after the PostgreSQL installation, to start it:
-
-```
-service postgresql start
-```
-
-If you want to have two separate machines, run these commands on the database server once you've installed the dependencies mentioned above on both servers:
+# If you want PostgreSQL
+export FEATURES=postgres
 
-```bash
-service postgresql start
-su - postgres
-createuser -d -P plume
-createdb -O plume plume
+# If you want SQlite
+export FEATURES=sqlite
 ```
 
-
 ## Running migrations
 
 Migrations are scripts used to update the database. They are run by a tool called Diesel, which can be installed with:
 
 ```bash
-cargo install diesel_cli --no-default-features --features postgres --version '=1.2.0'
+cargo install diesel_cli --no-default-features --features $FEATURES --version '=1.3.0'
 ```
 
-Plume should normally run migrations on your behalf as needed, but if you want to run them manually, use the following command:
+To run the migrations, you can do:
 
 ```bash
+# For a PostgreSQL database
 diesel migration run --database-url postgres://USER:PASSWORD@IP:PORT/DATABASE_NAME
+
+# For a SQlite database
+diesel migration run --database-url ./plume.sqlite3
 ```
 
-This command may be useful if you decided to use a separate database server.
+Migrations should be run before using Plume or the `plm` CLI tool, and after each update.
+When in doubt, run them.
 
-## Starting Plume
+## Running Plume
 
-First, you'll need to install Plume and the CLI tools to manage your instance.
+Then, you'll need to install Plume and the CLI tools to manage your instance.
 
 ```
-cargo install && cargo install --path plume-cli
+cargo install --no-default-features --features $FEATURES
+cargo install --no-default-features --features $FEATURES --path plume-cli
```

Comment:
@igalic :arrow_up:

Reply:
who needs build when you can just install 😅